docs/source/plugins/for_plugin_developers.rst

```diff
@@ -136,34 +136,35 @@ with :func:`numpy.save`)
 
 .. _hookimpl-decorator:
 
-Decorating your function with ``pluggy.HookimplMarker``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Decorating your function with ``HookimplMarker``
```

Comment:
Is it too early to discuss renaming this to `HookImplementationMarker`? ;)

Reply:
Or maybe even just `HookImplementation`, but I understand the utility of the word "marker".

Reply:
will do.  but can't be `HookImplementation` since that's already the thing that encapsulates an [instantiated hook implementation](https://napari-plugin-engine.readthedocs.io/en/latest/api.html#hookimpl) (not the decorator)
but I can change that too while I'm at it...  would be good if you can take a quick peek through the [new API Reference](https://napari-plugin-engine.readthedocs.io/en/latest/api.html#) and tell me if you want anything else changed.

Reply:
it's done!  long live the [`HookImplementationMarker`](https://napari-plugin-engine.readthedocs.io/en/latest/api.html#hookimplementationmarker)